docs/rules-engine/how-to-use/dedicated-component-ruleset.md

```diff
@@ -18,60 +63,40 @@ First, you need to have a ruleset defined in your json (created via the CMS inte
       "id": "e5th46e84-5e4th-54eth65seth46se8th2",
       "name": "the second ruleset",
       "linkedComponents": {
-        "or": [ // 'or' keyword means: if at least one component from the list is registered, the ruleset is active
+        // 'or' keyword means: if at least one component from the list is registered, the ruleset is active
+        "or": [ 
```

Comment:
Maybe add a note to say that the `and` is not supported but if needed they can open an issue (link to open the issue) and we will add the support